test/core.spec.ts

```diff
@@ -669,4 +682,54 @@ contract("Core", (accounts) => {
       });
     });
   });
+
+  describe("#create", async () => {
+    let factoryAddress: Address;
+
+
+    beforeEach(async () => {
+      await deployCoreAndInitializeDependencies();
+      mockToken = await coreWrapper.deployTokenAsync(ownerAccount);
+    });
+
+    async function subject(): Promise<string> {
+      return core.create.sendTransactionAsync(
+        factoryAddress,
+        [mockToken.address],
+        [ONE],
+        ONE,
+        "New Set",
+        "SET",
+        { from: ownerAccount },
+      );
+    }
+
+    describe("when the factory is valid", async () => {
```

Comment:
this should be baked into the subject as a default value, see: core#batchWithdraw.subject

Reply:
then you can get rid of this first describe